src/app/(proper_react)/(redesign)/(authenticated)/admin/announcements/AnnouncementsAdmin.tsx

```diff
@@ -156,315 +159,299 @@ export const AnnouncementsAdmin = (props: Props) => {
+                      >
+                        {notification.label}
+                      </div>
+                    </div>
+                  </li>
+                ))}
+                <button
+                  className={styles.addButton}
+                  onClick={() => setIsModalOpen(true)}
+                >
+                  + Add new announcement
+                </button>
+              </ul>
+            </div>
+
+            {/* Announcement Details */}
+            {activeAnnouncement && (
+              <div className={styles.notificationSettings}>
+                <h2>Details</h2>
+                <dl>
+                  <dt>Announcement ID</dt>
+                  <dd>{activeAnnouncement.announcement_id}</dd>
+
+                  <dt>Title</dt>
+                  <dd>
                     <span className={styles.missingLabelContainer}>
                       <LocalizedAnnouncementString
-                        announcement={notification}
+                        announcement={activeAnnouncement}
                         type="title"
                       />
                       <MissingFluentId
-                        announcement={notification}
+                        announcement={activeAnnouncement}
                         type="title"
                       />
                     </span>
-                  </p>
-                  <p className={styles.description}>
+                  </dd>
+
+                  <dt>Description</dt>
+                  <dd>
                     <span className={styles.missingLabelContainer}>
                       <LocalizedAnnouncementString
-                        announcement={notification}
+                        announcement={activeAnnouncement}
                         type="description"
                       />
                       <MissingFluentId
-                        announcement={notification}
+                        announcement={activeAnnouncement}
                         type="description"
                       />
                     </span>
-                  </p>
-                </div>
-                <div className={styles.pills}>
-                  <div
-                    className={`${styles.statusPill} ${styles[notification.label]}`}
+                  </dd>
+
+                  <dt>Small Image</dt>
+                  <dd>
+                    <ImageWithFallback
+                      src={smallImagePath}
+                      fallbackSrc="/images/announcements/fallback/small.svg"
+                      alt="Small Image"
+                      width={500}
+                      height={300}
+                      className={styles.smallImage}
+                    />
+                  </dd>
+
+                  {/* Big Image */}
+                  <dt>Big Image</dt>
+                  <dd>
+                    <ImageWithFallback
+                      src={smallImagePath}
```

Comment:
Should this be:
```suggestion
                      src={bigImagePath}
```
?